src/pages/register/index.js

```diff
@@ -9,17 +9,31 @@ const Register = () => {
   const { onRegister } = useAuth();
   const [formData, setFormData] = useState({ email: "", password: "" });
   const [errorMsg, setErrorMsg] = useState("");
+
   const onFormChange = (e) => {
     const { name, value } = e.target;
     setFormData({ ...formData, [name]: value });
   };
 
+  function validateEmail() {
```

Comment:
why are we not re-using these functions in the login?

you could extract these functions in a separate file: `/src/utils/validations.js`

Reply:
Doing that now